src/main/java/io/cdap/plugin/ariba/source/AribaBatchSource.java

```diff
@@ -87,7 +87,8 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
             getOutputSchema());
         }
       } catch (IOException | InterruptedException exception) {
-        failureCollector.addFailure(exception.getMessage(), null);
+        failureCollector.addFailure("Endpoint is incorrect. Unable to validate the source with the provided " +
+            "endpoint.", null);
```

Comment:
Why are we ignoring the exception message here ?

Reply:
Because the message thrown by exception is not user friendly.